recipes/eigen/all/conanfile.py

```diff
@@ -21,37 +24,42 @@ class EigenConan(ConanFile):
     }
     license = ("MPL-2.0", "LGPL-3.0-or-later")  # Taking into account the default value of MPL2_only option
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def configure(self):
         self.license = "MPL-2.0" if self.options.MPL2_only else ("MPL-2.0", "LGPL-3.0-or-later")
 
+    def layout(self):
+        cmake_layout(self)
```

Comment:
I advice to put source code in a dedicated subdir (I use `src` to keep folder short, to avoid long path issue on Windows), so that patches & source code are not mixed-up.

Reply:
Done.